app/javascript/dashboard/i18n/locale/en/chatlist.json

```diff
@@ -50,6 +50,9 @@
       },
       "sort_on_priority": {
         "TEXT": "Priority"
+      },
+      "sort_on_waiting_since": {
+        "TEXT": "Longest waiting"
```

Comment:
```suggestion
        "TEXT": "Oldest Unanswered"
```

A suggestion (feel free to ignore it) We can perhaps use a better copy. Longest waiting may not be very clear.

Reply:
Same suggestion. What about the text `Outstanding Replies`?

Reply:
"pending response" ? :D

Reply:
Let us go with "pending response" for now.